packages/dom/src/serialize-image-srcset.js

```diff
@@ -0,0 +1,33 @@
+// Return list of link of all srcset images to be captured
+export function serializeImageSrcSet({ dom }) {
```

Comment:
This requires extra queries on dom and extra iterations on dom. So far we have always tried to add to cloneNode function which anyway's does one tree walk on the node to avoid loading same nodes again. That could potentially give slight performance boost.

Reply:
resolved